src/main/java/connectors/consumer_producer/TrainDataDeserializer.java

```diff
@@ -0,0 +1,54 @@
+package connectors.consumer_producer;
+
+import com.google.gson.Gson;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.io.*;
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+
+public class TrainDataDeserializer implements Deserializer<TrainData> {
+
+    @Override
+    public void configure(Map<String, ?> configs, boolean isKey) {
+
+    }
```

Comment:
Perhaps we should add a comment that this method is necessary for the implementation of Deserializer so that it won't be refactored away in the future.